modules/order/src/OrderProcessorInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Drupal\commerce_order;
+
+use Drupal\commerce_order\Entity\OrderInterface;
+
+/**
+ * Defines an interface for processing an order.
```

Comment:
Maybe something like:
"Defines an interface for order processors.

Order processors modify the order during the refresh process."